metadata-ingestion/tests/integration/tableau/test_tableau_ingest.py

```diff
@@ -689,3 +694,35 @@ def test_tableau_signout_timeout(pytestconfig, tmp_path, mock_datahub_graph):
         sign_out_side_effect=ConnectionError,
         pipeline_name="test_tableau_signout_timeout",
     )
+
+
+def test_tableau_unsupported_csql():
+    config = mock.MagicMock()
```

Comment:
I would have used the config used in all other tests. Slightly preferable to using mock, as that saves us specifying configs like `stateful_ingestion.enabled` and `env` which are not directly related to test.
```

config = TableauConfig.parse_obj(config_source_default.copy())
config.extract_lineage_from_unsupported_custom_sql_queries = True
config.lineage_overrides = TableauLineageOverrides(
        database_override_map={"production database": "prod"}
    )
```

Reply:
Changed on [285be1a](https://github.com/datahub-project/datahub/pull/7561/commits/285be1ad666f6f5b19eeceae9091f242c11da043), though I still disabled stateful_ingestion to avoid an error trying to connect to the API. I could not figure an easy way of mocking that :/